### ISSUE TITLE
Feature request: command cooldowns

### ISSUE THREAD

**Issue by SlimeDog** [HoshiKurama/TicketManager#1]
Spigot 1.16.5
TicketManager 3.0.0

It would be nice to have command cooldowns, to prevent spamming the database. Default should be around 30, which will cause a miscreant to get bored before they can do significant damage. Something like:
```
# Command cooldown, measured in seconds:
command_cooldown: 30
```

**Comment by HoshiKurama** [comment]
Command cooldowns have been implemented in version 4.0.0 pre-release and work great! End users will be able to choose if players have a cooldown, the time of the cooldown if applicable, and there is also a cooldown override permission! I look forward to finishing initial testing of the pre-release, beta testing, writing documentation, and then sending out the full release!

~Hoshi

**Comment by SlimeDog** [comment]
I would be happy to test the pre-release.
